cmd/kots/cli/upstream-upgrade.go

```diff
@@ -99,9 +99,16 @@ func UpstreamUpgradeCmd() *cobra.Command {
 				}
 
 				imagesRootDir := filepath.Join(airgapRootDir, "images")
```

Comment:
this can be moved to be inside this if statement here https://github.com/replicatedhq/kots/pull/1714/files#diff-450f22c924117dc9b5e725632d49bd946038c4d9505484ee4c72a72f01fd168cR102